src/Microsoft.ML.Data/Prediction/Calibrator.cs

```diff
@@ -313,9 +340,12 @@ bool ISingleCanSaveOnnx.SaveAsOnnx(OnnxContext ctx, string[] outputNames, string
     }
 
     [BestFriend]
-    internal sealed class CalibratedPredictor : ValueMapperCalibratedPredictorBase, ICanSaveModel
+    internal sealed class CalibratedPredictor<TSubPredictor,TCalibrator> :
```

Comment:
>TSubPredictor [](start = 46, length = 13)

Same here please.

Reply:
Sure. I also replace `Predictor` from this class name.

---
In reply to: [255162847](https://github.com/dotnet/machinelearning/pull/2435#discussion_r255162847) [](ancestors = 255162847)